VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                              <locus target="#161vb"/>
+                              ይሰፍሑ፡ ክነፊሆሙ። ወይብሉ፡ ዝንቱ፡ ውእቱ፡ ንጉሠ፡ <choice><sic>ሰ</sic><corr resp="MV">ስ</corr></choice>ብሐት፡ መጽአ፡ ይሥረይ፡ ኃጢአተ፡ ዓለም፡ 
+                              በዕበየ፡ ሣህሉ። ሰ፡ ቅ፡ 
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.3">
+                           <locus from="161vb" to="164ra"/>
+                           <title type="complete" ref="LIT2509Weddas#Wednesday"/>
+                           <incipit xml:lang="gez">
+                              <locus from="161vb" to="162ra"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘረቡዕ፡</hi> ኵሉ፡ ሠራዊተ፡ ሰማያት፡ ይብሉ፡ ብፅዕት፡ ሰማይ፡ ዳግሚት፡ ዲበ፡ ምድር። ኆኅተ፡ ምሥራቅ፡ 
+                              <hi rend="rubric">ማርያም፡</hi> ድንግል፡ ከብካብ፡ ንጹ<pb n="162r"/><cb n="a"/>ሕ፡
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#164ra"/>
+                              ወአንሶሰወ፡ ምስለ፡ ሰብእ፡ እስመ፡ መሐረ፡ ውእቱ፡ ወመፍቀሬ፡ ሰብእ፡ አድኃነ፡ ነፍሳቲነ፡ በምጽአቱ፡ ቅዱስ፡ ሰ፡ ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.4">
+                           <locus from="164ra" to="166rb"/>
+                           <title type="complete" ref="LIT2509Weddas#Thursday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#164ra"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘሐሙስ፡ ዕፀ፡ እንተ፡</hi> ርእየ፡ ሙሴ፡ በነደ፡ እሳት፡ ውስተ፡ ገዳም፡ ወአዕፁቂሃ፡ ኢትውኢ፡ ትመስል፡ <hi rend="rubric">ማርያም፡</hi> 
+                              ድንግል፡ ዘእንበለ፡ ርኩስ።
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#166rb"/>
+                              ወተወልደ፡ በቤተ፡ ልሔም፡ በከመ፡ ሰበኩ፡ ነቢያት፡ አድኃነነ፡ ወቤዘወነ፡ ወረሰየነ፡ ሕዝበ፡ ዚአሁ፡ ሰ፡ ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.5">
+                           <locus from="166rb" to="167vb"/>
+                           <title type="complete" ref="LIT2509Weddas#Friday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#166rb"/>
+                              <hi rend="rubric"><surplus>ው</surplus>ው<add place="interlinear">ዳ</add>ሴ፡ </hi> ዘዓርብ፡ ቡርክት፡ አንቲ፡ እምአንስት፡ ወቡሩክ፡ ፍሬ፡ 
+                              ከር<supplied reason="omitted" resp="MV">ሥ</supplied>ኪ፡ ኦ<hi rend="rubric">ማርያም፡</hi> 
+                              ድንግል፡ ወላዲተ፡ አምላክ፡ ዘእንበለ፡ ርኵስ፡ ሠረቀ፡ ለነ፡ እምኔኪ፡ ፀሐየ፡ ጽድቅ።
+                            </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#167vb"/>
+                              በ<supplied reason="omitted" resp="MV">እ</supplied>ንተዝ፡ ንዌድሰኪ፡ በከመ፡ ገብርኤል፡ መልአክ፡ እንዘ፡ ንብል፡ ቡርክት፡ አንቲ፡  <hi rend="rubric">ማርያም፡</hi> 
```

Comment:
ho l'impressione che tra አንቲ፡  e  <hi rend="rubric">ማርያም፡</hi> ci sia un doppio spazio

Reply:
For Sanne: I have the impression that there is a double space between አንቲ፡ and ማርያም፡

Reply:
corrected anyway